zarr/zarr-in-practice.ipynb

```diff
@@ -8,11 +8,38 @@
     "\n",
```

Comment:
<p>Personally I would minimize stating specific suggestions on size for compressed or uncompressed chunks (although I do think 1MB uncompressed or greater is reasonable) as the best chunk shape and size can really depend on the intended use cases and storage system.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Yeah that makes sense. I'll take some of that extra text out.</p>